From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martijn Muijsers <martijnmuijsers@live.nl>
Date: Sat, 18 Feb 2023 21:27:07 +0100
Subject: [PATCH] Suki configuration

License: MIT (https://opensource.org/licenses/MIT)
Suki - https://github.com/SuCraft/Suki

This patch is based on the following patch:
"Timings v2"
By: Aikar <aikar@aikar.co>
As part of: Paper (https://github.com/PaperMC/Paper)
Licensed under: MIT (https://opensource.org/licenses/MIT)

diff --git a/src/main/java/org/bukkit/Server.java b/src/main/java/org/bukkit/Server.java
index 4c519dbd706cac728f31a7ccd99cc1e165812599..401ae80838bc4ecacdbb7796cc053251b4f0eb96 100644
--- a/src/main/java/org/bukkit/Server.java
+++ b/src/main/java/org/bukkit/Server.java
@@ -2012,6 +2012,14 @@ public interface Server extends PluginMessageRecipient, net.kyori.adventure.audi
         }
         // Gale end - Gale configuration - API
 
+        // Suki start - Suki configuration - API
+        @NotNull
+        public org.bukkit.configuration.file.YamlConfiguration getSukiConfig()
+        {
+            throw new UnsupportedOperationException("Not supported yet.");
+        }
+        // Suki end - Suki configuration - API
+
         // Paper end
 
         /**
